lib/UserInterface/textinput/Callbacks.h

```diff
@@ -30,7 +30,7 @@ namespace textinput {
                           EditorRange& R /*out*/,
                           std::vector<std::string>& DisplayCompletions /*out*/)
     = 0;
-    virtual ~TabCompletion();
+    virtual ~TabCompletion() {}
```

Comment:
ExceptionRTTI.cpp throws exceptions and it's built with RTTI.
That means anything that want's to catch what it throws must also be built with RTTI.
Linux seems to not care but both OS X and Windows do.

So when building UserInterface.cpp with RTTI, unless it sees ~TabCompletion, it will try to link to it, but ~TabCompletion was not built with RTTI so link will fail.

I've worked out a better solution, where users call into a RunLoop in ExceptionRTTI.cpp.
The RunLoop will solve any problems for users of cling that don't have exception handling as well.
But that change is much further down the road.